lib/atlas/initializer_input.rb

```diff
@@ -0,0 +1,23 @@
+module Atlas
+  class InitializerInput
+    include ActiveDocument
+
+    DIRECTORY = 'initializer_inputs'
+
+    attribute :query,           String
+    attribute :share_group,     Symbol
```

Comment:
> If you specify one input from a group, you have to specify all of them

No.

You can just alter a single `share` of a `share_group`. They don't have to add up to a 100%. The balancer inside of ETModel/ETEngine will take care of it in the front-end.  The amount of control you have is limited in that case but it's not invalid todo so.